src/ServantElm.hs

```diff
@@ -0,0 +1,231 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+
+module ServantElm
+  ( elmForAPI,
+  )
+where
+
+import Data.Maybe (catMaybes)
+import Data.Proxy (Proxy (..))
+import Data.Text as T (pack, takeWhile)
+import Elm (Elm (..))
+import Elm.Ast
+  ( ElmDefinition,
+    ElmPrim (..),
+    TypeName (TypeName, unTypeName),
+    TypeRef (..),
+    definitionToRef,
+  )
+import Lens.Micro ((^.))
+import Network.Wai ()
+import Prettyprinter
+  ( Doc,
+    comma,
+    dquotes,
+    encloseSep,
+    equals,
+    hsep,
+    indent,
+    lbrace,
+    lbracket,
+    line,
+    lparen,
+    parens,
+    pretty,
+    punctuate,
+    rbrace,
+    rbracket,
+    rparen,
+    space,
+    vsep,
+    (<+>),
+  )
+import Prettyprinter.Internal (unsafeTextWithoutNewlines)
+import Servant.Foreign
+  ( GenerateList,
+    HasForeign (Foreign),
+    HasForeignType (..),
+    PathSegment (unPathSegment),
+    Req,
+    Segment (unSegment),
+    SegmentType (Cap, Static),
+    camelCase,
+    listFromAPI,
+    path,
+    reqFuncName,
+    reqMethod,
+    reqReturnType,
+    reqUrl,
+  )
+import Prelude hiding ((<$>))
```

Comment:
You no longer need to hide this now that you've switched over to the other pretty printer.

Reply:
done